clients/java/src/main/java/com/google/myanmartools/Transliterate.java

```diff
@@ -0,0 +1,90 @@
+/* Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.myanmartools;
+
+import java.util.ArrayList;
+
+/**
+ * A Java engine for running the phases of transliteration, similar to the ICU4J
+ * transliterator.
+ */
+
+public class Transliterate {
+
+  private final ArrayList<Phase> translitPhases;
+  private final String name;  // For identification
+
+  private Boolean debugMode;  // Print details of execution.
+
+  // Creates a new transliterator.
+  public Transliterate() {
+    translitPhases = new ArrayList<>();
+    name = "";
+    debugMode = false;
+  }
+
+  public Transliterate(String id) {
+    translitPhases = new ArrayList<>();
+    name = id;
+  }
+
+  public Phase addPhase() {
+    Phase newPhase = new Phase();
+    newPhase.setInfo(" " + translitPhases.size());
+    translitPhases.add(newPhase);
+    newPhase.setDebugMode(debugMode);
+    return newPhase;
+  }
+
+  public void setDebugMode(Boolean newMode) {
```

Comment:
make this package-private.  If a user is actually debugging, they probably are hacking in the library locally and don't need a public method.

Reply:
Done